src/metabase/api/setup.clj

```diff
@@ -148,9 +149,9 @@
       :link        "/admin/datamodel/database"
       :completed   has-hidden-tables?
       :triggered   (>= num-tables 20)}
-     {:title       "Organize questions"
-      :group       "Curate your data"
-      :description "Have a lot of saved questions in Metabase? Create collections to help manage them and add context."
+     {:title       (tru "Organize questions")
+      :group       (tru "Curate your data")
+      :description (tru "Have a lot of saved questions in {0}? Create collections to help manage them and add context." (tru "Metabase"))
```

Comment:
can't someone translate 

```
Have a lot of saved questions in Metabase? Create collections to help manage them and add context.
```

as-is?

Reply:
e.g. you have `Loading Metabase...` later on without making `Metabase` a param

Reply:
`Metabase` will need to be a parameter for whitelabeling where you can change it to something else.